src/win32/agent_auth.c

```diff
@@ -473,11 +474,26 @@ int main(int argc, char **argv)
     printf("INFO: Using agent name as: %s\n", agentname);
 
     // Send request
+    char *secure_msg;
```

Comment:
I suggest changing the name of this variable to **enrollment_msg** and always initialize a pointer to NULL.

Reply:
Fixed!